src/charts/raster-chart.js

```diff
@@ -170,6 +170,28 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     _layerNames = {}
   }
 
+  _chart.removeLayer = function(layerName) {
+    const layer = _layerNames[layerName]
+    if (layer && layer.destroyLayer && typeof layer.destroyLayer === 'function') {
+      layer.destroyLayer(_chart)
+    }
+    delete _layerNames[layerName]
+    _layers = _layers.filter(name => layerName !== name)
+    return layer
+  }
+
+  _chart.unshiftLayer = function(layerName, layer) {
```

Comment:
@vastcharade Also added this, which adds a layer at the front of the array instead of pushing onto the end. This allows more control over layer display ordering in immerse.